### PR TITLE
Add username from nextauth

### LINKED ISSUE #153: nav bar: take username from nextauth
image
Expected: should show my username

### PR DESCRIPTION
**Pull request by sltsheryl**
Logged in with my google account

Screenshot 2023-01-10 at 2 46 13 AM

Fixes #153

### CODE REVIEW

**Review by MorningLit — CHANGES_REQUESTED** [review]
For the component Dropdown can you change the ones highlighted?
image
-For key, it should always be a unique key, in this case u can pass in item.clickOption
image
i think just having the bg be that color and not when it is hovered, is better

**Review by MorningLit — COMMENTED** [review]
20230111133506_rec_
Also can you remove the w-[775px] from all 3 navbars?

20230111133403_rec_
So that we can achieve this (Up to you whether you wan to fix the first navbar, i can close an eye on it)

**Comment by sltsheryl** [comment]
> So that we can achieve this (Up to you whether you wan to fix the first navbar, i can close an eye on it)

Should we collapse the nav bar like this? Or leaving it as the next row is better?
Screenshot 2023-01-12 at 2 22 58 AM

Currently, the new one is like this
Screenshot 2023-01-12 at 2 23 59 AM

**Comment by MorningLit** [comment]
@sltsheryl if you want you can go ahead, but I don't think there is figma design for it right(i think, if not I can't seem to find it).
And also I believe we aren't supporting mobile so we should only concern ourselves with 1024px (laptop min display resolution)
image

File: components/navbar/DropDown.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const DropDown = ({ buttonName, dropdownItems }) => {
       <Menu.Items className={`absolute mt-8 divide-y divide-gray-100 shadow`}>
         {arr.map(item => {
           return (
-            <Menu.Item as='div' key='item' className={`block py-2 px-4 hover:bg-gray-100`}>
+            <Menu.Item as='div' key='item.clickOption' className={`block py-2 px-4`}>
```
Comment (MorningLit) [inline components/navbar/DropDown.tsx:16]: Not the string 'item.clickOption' but {item.clickOption} instead

**Review by MorningLit — CHANGES_REQUESTED** [review]

File: components/navbar/DropDown.tsx (hunk 1 above)
Comment (MorningLit) [reply components/navbar/DropDown.tsx:16]: and i said remove the hover part only, not the entire bg-gray-100 😭

Comment (sltsheryl) [reply components/navbar/DropDown.tsx]: > Not the string 'item.clickOption' but {item.clickOption} instead

Oh yea sry

Comment (sltsheryl) [reply components/navbar/DropDown.tsx]: > and i said remove the hover part only, not the entire bg-gray-100 😭

Okay for easy reference, this is what's given in Figma ⬇️  I went to follow this
Screenshot 2023-01-13 at 4 01 56 PM

**Review by MorningLit — APPROVED** [review]

**Pull request merged**